mappings/net/minecraft/item/EnchantedBookItem.mapping

```diff
@@ -8,5 +8,13 @@ CLASS net/minecraft/class_1428 net/minecraft/item/EnchantedBookItem
 		ARG 2 list
 	METHOD method_4609 getAsItemStack (Lnet/minecraft/class_1134;)Lnet/minecraft/class_1071;
 		ARG 1 info
+	METHOD method_4610 getLootTable (Ljava/util/Random;)Lnet/minecraft/class_847;
+		ARG 1 random
 	METHOD method_4611 getEnchantmentNbt (Lnet/minecraft/class_1071;)Lnet/minecraft/class_474;
 		ARG 0 stack
+		ARG 1 itemStack
```

Comment:
Wouldn't changing  ARG 1 to `stack` interfere with ARG 0? I'm not able to see ARG 0 in Enigma, but it redlines when I try to rename the ItemStack to `stack`.

Reply:
> Wouldn't changing ARG 1 to `stack` interfere with ARG 0? I'm not able to see ARG 0 in Enigma, but it redlines when I try to rename the ItemStack to `stack`.

yes it would, yes I am stupid